test/test_unistd.rs

```diff
@@ -466,9 +465,7 @@ fn test_fchown() {
     fchown(fd, uid, gid).unwrap();
     fchown(fd, uid, None).unwrap();
     fchown(fd, None, gid).unwrap();
-
-    mem::drop(path);
-    fchown(fd, uid, gid).unwrap_err();
+    fchown(999999999, uid, gid).unwrap_err();
```

Comment:
what's this change doing?

Reply:
Here's what I put in the commit message:

Fix a race in the test_unistd::test_fchown test
    
The test was assuming that once dropped, a temporary file's file
descriptor would be invalid.  But it might not be, because another file
might be opened with the same file descriptor.  Instead, use an
obviously invalid file descriptor.

Reply:
ah my bad: didn't look at the individual commits. sorry!